src/Orleans.Core/Configuration/Options/ConsistentRingOptions.cs

```diff
@@ -0,0 +1,12 @@
+
+namespace Orleans.Hosting
+{
+    public class ConsistentRingOptions
```

Comment:
I think this class should be in `orleans.runtime`, where it is used and where `VirtualBucketsRingProvider` lives